tests/prettier/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -13,6 +13,100 @@ function fn() {
 "
 `;
 
+exports[`test jsx-split-attrs.js 1`] = `
+"long_closed =
+  <BaseForm url=\"/auth/google\" method=\"GET\" colour=\"blue\" size=\"large\" submitLabel=\"Sign in with Google\"/>
+
+long_open =
+  <BaseForm url=\"/auth/google\" method=\"GET\" colour=\"blue\" size=\"large\" submitLabel=\"Sign in with Google\">
+    hello
+  </BaseForm>
+
+short_closed =
+  <BaseForm url=\"/auth/google\" method=\"GET\"/>
+
+short_open =
+  <BaseForm url=\"/auth/google\" method=\"GET\">
+    hello
+  </BaseForm>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+long_closed = <BaseForm
+  url=\"/auth/google\"
+  method=\"GET\"
+  colour=\"blue\"
+  size=\"large\"
+  submitLabel=\"Sign in with Google\"
+/>;
+
+long_open = <BaseForm
```

Comment:
Hmm... I should probably refactor this (and any others like the corresponding code in `ReturnStatement`) within `JSXElement` itself... might be tricky...